web-access-app/src/main/java/edu/unc/lib/boxc/web/access/controllers/FullRecordController.java

```diff
@@ -225,6 +227,24 @@ public String getFullRecord(String pidString, Model model, HttpServletRequest re
             String faUrl = findingAidUrlService.getFindingAidUrl(collectionId);
             model.addAttribute("collectionId", collectionId);
             model.addAttribute("findingAidUrl", faUrl);
+
+            // Get digital exhibits found on a collection
+            ContentObjectRecord exhibitObj = briefObject;
```

Comment:
this controller method is so long, maybe we should put the digital exhibits stuff into a private method?